src-electron/db/query-package.js

```diff
@@ -339,7 +357,22 @@ async function updatePathCrc(db, path, crc, parentId) {
   return dbApi.dbUpdate(
     db,
     'UPDATE PACKAGE SET CRC = ? WHERE PATH = ? AND PARENT_PACKAGE_REF = ?',
-    [path, crc, parentId]
+    [crc, path, parentId]
+  )
+}
+
+/**
+ * Updates a is in sync in the package table.
+ * @param {*} db
+ * @param {*} packageRef
+ * @param {*} isInSync
+ * @returns Promise of an update.
+ */
+async function updatePackageIsInSync(db, packageRef, isInSync) {
+  return dbApi.dbUpdate(
+    db,
+    'UPDATE PACKAGE SET IS_IN_SYNC = ? WHERE PACKAGE_ID = ?',
+    [isInSync, packageRef]
```

Comment:
Should we not use `dbApi.toDbBool()` here?
I am always cautious about taking arguments from JS and turning them into DB boolean values (0/1).

Reply:
Good point.